### PR TITLE
docs: add a kb article about setting up zot mirroring

### PR DESCRIPTION
**Pull request by mbshields**
**What type of PR is this?**
documentation

**Which issue does this PR fix**:
explains how to configure mirroring

**What does this PR do / Why do we need it**:

**If an issue # is not available please add repro steps and logs showing the issue**:

**Testing done on this change**:

**Automation added to e2e**:

**Will this break upgrades or downgrades?**

**Does this PR introduce any user-facing change?**:

```release-note

```

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: docs/articles/mirroring.md [hunk 1]
```diff
@@ -0,0 +1,432 @@
+# OCI Registry Mirroring With zot
+
+> :point_right: A `zot` registry can mirror one or more upstream OCI registries.
```
Comment (rchincha) [inline docs/articles/mirroring.md:3]: ..., including popular cloud registries such as dockerhub and gcr.

@peusebiu pls. confirm

File: docs/articles/mirroring.md [hunk 2]
```diff
@@ -0,0 +1,432 @@
...
+    "repositories":[
+      "docker-images/alpine",
+      "k8s-images/kube-proxy"
+    ]
+  }
+``` 
+
+## Example: Multiple registries with mixed mirroring modes
+
+<details>
```
Comment (rchincha) [inline docs/articles/mirroring.md:279]: Only this example has a dropdown?

File: docs/articles/mirroring.md [hunk 3]
```diff
@@ -0,0 +1,432 @@
...
+    }
+  }
+}
+```
+</details>
+
+[//]: # (https://github.com/project-zot/zot/blob/main/examples/config-sync.json)
+
+
+## Example: Support for subpaths in local storagge
```
Comment (rchincha) [inline docs/articles/mirroring.md:365]: "storagge" typo

File: docs/articles/mirroring.md [hunk 4]
```diff
@@ -0,0 +1,432 @@
...
+            {
+              "prefix": "**", 
+              "destination": "/k8s-images"
+            }
+          ],
+          "onDemand": true,
+          "tlsVerify": true
+        },
+        {
+          "urls": ["https://docker.io/library"],
```
Comment (rchincha) [inline docs/articles/mirroring.md:242]: A small example calling out docker.io sync explicitly pls.
^ @peusebiu

File: docs/articles/mirroring.md (hunk 1 above)
Comment (mbshields) [reply docs/articles/mirroring.md:3]: Is this OK?  ", including popular cloud registries such as Docker Hub and Google Container Registry (gcr.io)."

File: docs/articles/mirroring.md (hunk 2 above)
Comment (mbshields) [reply docs/articles/mirroring.md:279]: Sorry, I thought I removed them all.  What are people's opinions on this show/hide function for long config structures?

File: docs/articles/mirroring.md (hunk 3 above)
Comment (mbshields) [reply docs/articles/mirroring.md:365]: Fixed, thanks.

File: docs/articles/mirroring.md (hunk 4 above)
Comment (peusebiu) [reply docs/articles/mirroring.md:242]: I did that below, right?
```skopeo copy --src-tls-verify=false docker://localhost:8080/docker-images/alpine <dest>```

Comment (mbshields) [reply docs/articles/mirroring.md:242]: I can make it more explicit in that example like this:
"...causes zot to synchronize the content with the docker.io registry:"

And the next part as:
"...causes zot to synchronize the content with the gcr.io registry:"

File: docs/articles/mirroring.md (hunk 2 above)
Comment (mbshields) [reply docs/articles/mirroring.md:279]: Removed that dropdown, too.  Do people prefer the show/hide function for long config blocks?

**Review by rchincha — APPROVED** [review]
lgtm

**Pull request merged**